synapse/cryotank.py

```diff
@@ -1,16 +1,26 @@
-import lmdb
+import os
 import shutil
 import struct
 import logging
+import threading
+import contextlib
+from collections import namedtuple, defaultdict
 
+import lmdb  # type: ignore
+
+import synapse.exc as s_exc
 import synapse.glob as s_glob
 import synapse.common as s_common
-import synapse.eventbus as s_eventbus
-
 import synapse.lib.cell as s_cell
+import synapse.lib.lmdb as s_lmdb
 import synapse.lib.const as s_const
+import synapse.lib.queue as s_queue
 import synapse.lib.config as s_config
+import synapse.eventbus as s_eventbus
+import synapse.datamodel as s_datamodel
 import synapse.lib.msgpack as s_msgpack
+import synapse.lib.threads as s_threads
+import synapse.lib.datapath as s_datapath
```

Comment:
style nit  - we've normally sorted imports by level as well - https://github.com/vertexproject/synapse/blob/master/synapse/lib/cell.py#L11 is a good example

Reply:
ok